internal/commands/pull/inputs_test.go

```diff
@@ -7,6 +7,8 @@ import (
 	"path/filepath"
 	"testing"
 
+	"github.com/Netflix/go-expect"
```

Comment:
there's going to need to be a JIRA to fix the whole codebase before we start adopting this import pattern

ive no strong objections (but also no strong inclination) to do so.  feel free to file it, but otherwise lets keep things consistent for now and move this below the project imports